content/src/service/access/AccessCheckerForWearables.ts

```diff
@@ -128,9 +128,9 @@ export class AccessCheckerForWearables {
           const entries = Array.from(content?.entries() ?? [])
           const contentAsJson = entries.map(([key, hash]) => ({ key, hash })).sort(compare)
           const buffer = Buffer.from(JSON.stringify({ content: contentAsJson, metadata }))
-          return Hashing.calculateBufferHash(buffer)
+          return Promise.all([Hashing.calculateBufferHash(buffer), Hashing.calculateIPFSHash(buffer)])
```

Comment:
Don't you have the entity version here so you can only execute the necessary algorithm?

Reply:
But this hash is stored on the blockchain by the committee. I'm not sure which version they'll use, and it won't depend on the entity version. That's why we are checking for both